components/carousel-indicators/CarouselIndicators.tsx

```diff
@@ -0,0 +1,16 @@
+import clsx from 'clsx'
+import { View } from 'react-native'
+
+interface Props {
+  getPager: number
+  passKey: number
+}
+
+const CarouselIndicators = ({ getPager, passKey }: Props) => {
```

Comment:
cambiar declaracion de componente y nombre de carpeta, en su lugar : 
```suggestion
function CarouselIndicators({ getPager, passKey }: Props) {

components/carousel/CarouselIndicators.tsx
```